src/documents/management/commands/convert_thumbnails.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import shutil
+import tempfile
+import time
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+from documents.models import Document
+from documents.parsers import run_convert
+
+
+logger = logging.getLogger("paperless.management.convert_thumbnails")
+
+
+class Command(BaseCommand):
+
+    help = """
+        Converts existing PNG thumbnails into
+        WebP format.
+    """.replace(
+        "    ",
+        "",
+    )
+
+    def handle(self, *args, **options):
+
+        self.stdout.write("Converting all PNG thumbnails to WebP")
+
+        start = time.time()
+
+        documents = Document.objects.all()
+
+        with tempfile.TemporaryDirectory() as tempdir:
+
+            for document in documents:
+                existing_thumbnail = Path(document.thumbnail_path).resolve()
+
+                if existing_thumbnail.suffix == ".png":
+
+                    self.stdout.write(f"Converting thumbnail: {existing_thumbnail}")
+
+                    # Change the existing filename suffix from png to webp
+                    converted_thumbnail_name = existing_thumbnail.with_suffix(
+                        ".webp",
+                    ).name
+
+                    # Create the expected output filename in the tempdir
+                    converted_thumbnail = (
+                        Path(tempdir) / Path(converted_thumbnail_name)
+                    ).resolve()
+
+                    try:
+                        # Run actual conversion
+                        run_convert(
+                            density=300,
+                            scale="500x5000>",
+                            alpha="remove",
+                            strip=True,
+                            trim=False,
+                            auto_orient=True,
+                            input_file=f"{existing_thumbnail}[0]",
+                            output_file=str(converted_thumbnail),
+                        )
+
+                        if converted_thumbnail.exists():
+                            # Copy newly created thumbnail to thumbnail directory
+                            shutil.copy(converted_thumbnail, existing_thumbnail.parent)
+
+                            # Remove the PNG version
+                            existing_thumbnail.unlink()
+
+                            self.stdout.write(
+                                self.style.SUCCESS(
+                                    "Conversion to WebP completed",
```

Comment:
`converted_thumbnail` in this case will be a mostly random temporary directory, and the whole directory will be removed after the command completes.  So that doesn't seem relevant or useful to a user as they can't access the file after completion.

Reply:
These logs are not useful for the users per se: they will never want to look at or understand these.

However, these logs are really useful for bug reports when something goes wrong - and arguably having more information that might help to find problems is better than a static message.

Reply:
When you view the lines before its clear which youre talking about, seems OK to me as-is